src/ensembl/src/shared/radio/Radio.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import defaultStyles from './Radio.scss';
+
+export type RadioOption = {
+  value: string | number | boolean;
+  label: string;
+};
+
+export type RadioOptions = RadioOption[];
+
+type Props = {
+  onChange: (selectedOption: string | number | boolean) => void;
+  classNames?: any;
+  radioOptions: RadioOptions;
+  selectedOption: string;
```

Comment:
if radioOptions can be string, number, or boolean, then why is the selectedOption a string?

Reply:
Well spotted. I've updated the type.